src/main/java/com/instaclustr/sstabletools/cassandra/CassandraBackend.java

```diff
@@ -41,9 +44,11 @@ public static CassandraProxy getInstance() {
     private CassandraBackend() {}
 
     public List<String> getKeyspaces() {
-        return Schema.instance.getNonLocalStrategyKeyspaces()
-                     .stream()
-                     .map(ksmd -> ksmd.name).sorted().collect(Collectors.toList());
+        return Schema.instance.distributedKeyspaces()
+                .stream()
```

Comment:
**[Required]** You're missing a filter to exclude local strategy keyspaces here:
```java
return Schema.instance.distributedKeyspaces()
	.stream()
	.filter(keyspace -> keyspace.params.replication.klass != LocalStrategy.class)
	...
```